README.md

```diff
@@ -0,0 +1,45 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+
+## Built With
+
+- Javascript
+- Html
+- CSS
+- Webpack
+
+## App description
+It is a website where you can add different task to your To Do List.
+
+
+## Pre requisites
+have npx installed
+
+## Getting Started
+Get a copy of the repository running this command in your OS terminal: `git clone https://github.com/disc3110/to-do-list.git` to get a copy of the project.
+
+
+go to the project folder
+
+run `npm run start` 
```

Comment:
[OPTIONAL[

I strongly recommend to fix this
- After going to the project folder and running `npm run start` would give an error. You have missed one command to include which is `npm install`. Please mention it in your README file.